### ISSUE TITLE
MacBook started making sound

### ISSUE THREAD

**Issue by harshiempire** [pieces-app/support#309]
### Software

Pieces OS

### Operating System / Platform

macOS

### Your Pieces OS Version

10.0.0

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

So I have just started the Pieces OS and Pieces for developers and then I was using copilot, at that time Pieces OS was using 3.1 GB of my memory and then the MacBook Air M2 has started making keek keek sound I closed it. The sound stopped and then I started it again and the same sound has started. And for the first time in a year my Mac has started making sounds

**Comment by sam-at-pieces** [comment]
@harshiempire this sounds very frustrating, thank you for reaching out with the issue. Resolving this is high priority for me. Would you be able to let us know a couple of things to help us out?
- what foundation model were you using for your copilot chat? In this image I'm using gpt 4 turbo.
<img width="564" alt="Screenshot 2024-07-15 at 15 59 14" src="https://github.com/user-attachments/assets/66c89500-fab2-44d6-b515-cd24d0f8a148">
- were you using a specific chat pipeline when this occurred?
<img width="564" alt="Screenshot 2024-07-15 at 15 58 08" src="https://github.com/user-attachments/assets/d1ddb01c-5613-4f6a-9c92-c0da9809d236">
- had you added any relevant context to the chat and if so how large was the folder/files?
Thanks again for reaching out, we'll get this resolved as soon as possible!

**Comment by harshiempire** [comment]
First I had the work stream pattern engine working I guess thats what its called, the thing I was doing was giving a json to the GPT 4o can generating the answer just to see the speed of the responses compared to the chatgpt 4o I have. So the sound started after some back and fro and the sound started. Prior to that I was using the live context and uploaded a folder of the django project it was working fine I guess didn't look at the activity monitor I didn't get any sound back then, it was just the previous day. Right now the pieces os is just on and I am getting the same sound 

wait wait its funny I turned off the workstream thing and there is no sound now 😂wow so that seems to be the problem while I had the chat going on with the folder of django the other day I had no sound and didnt know that this has existed, I saw one of your mail and then the yt video. It was next day I turned it on 

I hope this is helpful thanks for responding
